source/components/composing-components.md

```diff
@@ -10,6 +13,8 @@ Just like we compose regular HTML elements, we can do the same with components.
 {{/user-list}}
 ```
 
+First, a review of how to use components.
```

Comment:
Composing components is the very last item in the TOC for components. I'll attempt to ponder something else appropriate, open to ideas.